### DISCUSSION TITLE
quake-mode - start minimized/no focus and preventing exit via exit-command

### DISCUSSION THREAD

**Discussion by toraxmalu** [microsoft/terminal#12865] [Ideas]
Hi,

the quake window can be a useful option for a quick intermediate terminal session. but two points would improof the feature:
* protect against an exit via exit
* start option to start the window closed / minimized.

While a start via `cmd /c start /min wt.exe -w _quake` (PowerShell with `Start-Process`) on a system with sigle screen or main screen the outmost left produces a funny small console window, the windows terminal resizes correct, when there are other screens left of the main screen.

Thanks.

Cheers
Torax

**Comment by zadjii-msft** [comment]
Thanks for the suggestion! This is actually already being tracked by another issue on our repo - please refer to #9996 for more discussion.

/dup #9996
